File: fabric/src/main/java/net/impleri/playerskills/client/fabric/PlayerSkillsClientFabric.java
package net.impleri.playerskills.client.fabric;

import net.fabricmc.api.ClientModInitializer;
import net.impleri.playerskills.client.PlayerSkillsClient;

public class PlayerSkillsClientFabric implements ClientModInitializer {
    @Override
    public void onInitializeClient() {
        PlayerSkillsClient.init();
    }
}

File: fabric/src/main/kotlin/net/impleri/playerskills/client/fabric/PlayerSkillsClientFabric.kt
package net.impleri.playerskills.client.fabric

import net.fabricmc.api.ClientModInitializer
import net.impleri.playerskills.client.PlayerSkillsClient

class PlayerSkillsClientFabric : ClientModInitializer {
  override fun onInitializeClient() {
    PlayerSkillsClient.init()
  }
}
